bindings/rust/bench/src/harness.rs

```diff
@@ -118,28 +128,26 @@ macro_rules! test_tls_bench_harnesses {
     $(
         mod $lib_name {
             use super::*;
+            use CipherSuite::*;
+            use ECGroup::*;
+            use HandshakeType::*;
 
             #[test]
-            fn test_handshake() {
-                let mut harness = <$harness_type>::default().unwrap();
-                assert!(!harness.handshake_completed());
-                harness.handshake().unwrap();
-                assert!(harness.handshake_completed());
-                assert!(harness.negotiated_tls13());
-            }
-
-            #[test]
-            fn test_different_crypto_config() {
-                use CipherSuite::*;
-                use ECGroup::*;
-
+            fn test_handshake_config() {
                 let (mut harness, mut crypto_config);
```

Comment:
Okay, this wasn't even a change in this PR so it's on me for missing it the first, time, but it feels like crypto_config doesn't need to be mutable here. Can we move the declaration inside the loop?